projects/robots/epson/scaraEpsonT6/protos/scaraEpsonT6.proto

```diff
@@ -0,0 +1,632 @@
+#VRML_SIM R2022b utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# Model of the T6-602S Epson Scara robot.
+# template language: javascript
+
+PROTO scaraEpsonT6 [
+  field SFVec3f           translation     0 0 0.01        # Is `Transform.translation`.
+  field SFRotation        rotation        0 0 1 0         # Is `Transform.rotation`.
+  field SFString          name            "scaraEpsonT6"  # Is `Solid.name`.
+  field SFString          controller      ""              # Is `Robot.controller`.
+  field MFString          controllerArgs  []              # Is `Robot.controllerArgs`.
+  field SFBool            supervisor      FALSE           # Is `Robot.supervisor`.
+  field SFBool            synchronization TRUE            # Is `Robot.synchronization`.
+  field SFString          customData      ""              # Is `Robot.customData`.
+  field SFString          window          ""              # Is `Robot.window`.
+  field SFBool            staticBase      FALSE           # Defines if the scara base should be pinned to the static environment.
+  field MFNode            handSlot        []              # Extends the shaft with new nodes in the hand slot.
+
+]
+{
+Robot {
+  translation IS translation
+  rotation IS rotation
+  synchronization IS synchronization
+  name IS name
```

Comment:
The anchor of the first joint should be where the arm actually revolves instead of being on the ground

![industrial_example_2](https://user-images.githubusercontent.com/44834743/154436921-ad2162a3-a665-451a-91f7-ee8b5f57d730.png)